mypy_requirements.txt

```diff
@@ -0,0 +1,2 @@
+mypy==0.770
+git+git://github.com/numpy/numpy-stubs.git@master
```

Comment:
In these very early stages it seems fair to pull from the master branch.

Reply:
+1